src/main/java/es/upm/etsisi/DefaultParameters.java

```diff
@@ -0,0 +1,94 @@
+package es.upm.etsisi;
+
+import es.upm.etsisi.models.Administrator;
+import es.upm.etsisi.models.User;
+
+import java.util.LinkedList;
+import java.util.Iterator;
+import java.util.Random;
+import java.text.Normalizer;
+
+public class DefaultParameters {
+    private static final int DEFAULT_PLAYERS = 6;
+    private static final int DEFAULT_TEAMS = 2;
+    private static final String[] DEFAULT_SURNAMES = { "Pérez", "Sánchez", "Jiménez", "Torres", "Blanco", "Navarro",
+            "Paredes", "Moreno" };
+    private static final String[] DEFAULT_NAMES = { "Luisa", "Manuel", "Kurt", "Sofia", "Robert", "Jose", "Ramón",
+            "Pablo" };
+    private static final String[] DEFAULT_TEAMNAMES = { "Alfa", "Beta", "Delta", "Omega", "Theta" };
+
+    public DefaultParameters() {
+    }
+
+    public static Administrator getDefaultAdmin() {
+        return new Administrator("admin@upm.es", "admin");
+    }
+
+    public static LinkedList<LinkedList<String>> getDefaultParticipants() {
+        LinkedList<LinkedList<String>> parameters = new LinkedList<>();
+        int j = 0;
+        for (int i = 0; i < DEFAULT_PLAYERS; i++) {
+            LinkedList<String> playerParams;
+            do {
+                playerParams = generatePlayer();
+            } while (parameters.contains(playerParams));
+            parameters.add(playerParams);
+            if (j < DEFAULT_TEAMS) {
+                LinkedList<String> teamParams;
+                do {
+                    teamParams = generateTeam(playerParams.get(1));
+                } while (!isValidTeam(parameters, teamParams));
+                parameters.add(teamParams);
+                j++;
+            }
+        }
+        return parameters;
+    }
+
+    private static boolean isValidTeam(LinkedList<LinkedList<String>> parameters, LinkedList<String> teamParameters) {
+        Iterator<LinkedList<String>> iterator = parameters.iterator();
+        boolean result = true;
+        while (iterator.hasNext() && result) {
+            LinkedList<String> currentItem = iterator.next();
+            if (!User.isUpmEmail(currentItem.get(0))) {
+                if (currentItem.get(0) == teamParameters.get(0) || currentItem.get(1) == teamParameters.get(1)) {
+                    result = false;
+                }
+            }
+        }
+        return result;
+    }
```

Comment:
Is this needed? Should you be able to generate invalid teams? Remember you cant add a player to a team if it already has it, it won't add it. Think about it.

Reply:
Since team creation depends solely on teamName and playerName, it lead to sometimes creating invalid parameters. This should not be necessary with having players identified by their DNI as a key.

I'll make it work with the new player handling when it's implemented #11